CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # v1.16
 
+## Release v1.16.5 - 2024/03/27
+
+### Bug Fix
+
+* Buffer: Fix emit error of v1.16.4 sometimes failing to process large data
+  exceeding chunk size limit
+  https://github.com/fluent/fluentd/pull/4447
```

Comment:
Are you sure that this is the really the cause of this issue?
If you aren't sure yet, the description should only mention about missing lock.

Reply:
I think as https://github.com/fluent/fluentd/issues/4446#issuecomment-2022037316.
This can cause emit error on the codes, but I have not confirmed it in the actual behavior yet.

Reply:
Confirmed: https://github.com/fluent/fluentd/issues/4446#issuecomment-2022134914